fourier/fourier_test.go

```diff
@@ -214,6 +214,153 @@ func TestCmplxFFT(t *testing.T) {
 	})
 }
 
+func TestDCT(t *testing.T) {
+	const tol = 1e-10
+	t.Run("NewDCT", func(t *testing.T) {
+		for n := 2; n <= 200; n++ {
+			dct := NewDCT(n)
+
+			want := make([]float64, n)
+			for i := range want {
+				want[i] = rand.Float64()
```

Comment:
These should have their own RNG in the test. Also, should these maybe use NormFloat so there are both positive and negative numbers, as well as numbers > 1? Sorry, missed this in the last PR.

Reply:
Same for the other tests

Reply:
Done for the source, but changing the norm floats causes failure that I don't have time to look into right now.